src/spec/controllers/functions_controller_spec.rb

```diff
@@ -1,7 +1,11 @@
 require 'rails_helper'
 
 RSpec.describe FunctionsController, type: :controller do
-  let(:valid_attributes) { build(:function).attributes }
+  let!(:tags_count) { 10 }
```

Comment:
I see you didn't need to change `tags_count` anywhere, so let's just use it in line 5 directly instead of having it as a `let` here